### ISSUE TITLE
Other language

### ISSUE THREAD

**Issue by upup666** [brianmatzelle/Chat.tv#5]
Is it possible to use other language than English?

**Comment by brianmatzelle** [comment]
Yes, the program relies on OpenAI's API models, so comprehension will depend on GPT 3.5s comprehension ability in whatever language you're speaking. 

If you want to use a different large language model (perhaps one that is trained in your language), change the function: chatgpt_query() in bot.py.

Let me know if you have a specific implementation request, I wouldn't have a problem adding and updating the repo.

**Comment by upup666** [comment]
Thank you for answer, gpts 3.5 works fine in Russian, so what setting i need to change so it could work in my language?

**Comment by upup666** [comment]
And also it using not openai api i tried it to get work but Everytime some errors, maybe its possible to update it to use openai api?

**Comment by upup666** [comment]
> Yes, the program relies on OpenAI's API models, so comprehension will depend on GPT 3.5s comprehension ability in whatever language you're speaking. 
> 
> 
> 
> If you want to use a different large language model (perhaps one that is trained in your language), change the function: chatgpt_query() in bot.py.
> 
> 
> 
> Let me know if you have a specific implementation request, I wouldn't have a problem adding and updating the repo.

Hi there, thanks for your answer. Chatgpt 3.5 works fine with my language

And also it using not openai api i tried it to get work but Everytime some errors, maybe its possible to update it to use openai api?

**Comment by brianmatzelle** [comment]
Currently developing a patch.

**Comment by upup666** [comment]
> Currently developing a patch.

I tried my best, and it works, but it answers to me in very long questions and also some strange answers)

**Comment by upup666** [comment]
> Currently developing a patch.

Hi there, when you can release it?)

**Comment by upup666** [comment]
Hi there how is it going? With update?
